apps/src/utils.js

```diff
@@ -110,7 +110,9 @@ exports.stripQuotes = function(inputString) {
  * Defines an inheritance relationship between parent class and this class.
  */
 Function.prototype.inherits = function (parent) {
-  this.prototype = _.create(parent.prototype, { constructor: parent });
+  this.prototype = Object.create(parent.prototype);
+  this.prototype.constructor = this;
+  this.superPrototype = parent.prototype;
```

Comment:
Oh, I see what you're saying - let people handle it on a case-by-case basis?  That's also fine - obviously a lot of classes don't need to explicitly invoke super methods, and they'll always have their superclass constructor in scope so they can call MyBaseClass.prototype.method.call()... which I guess is no more painful than MyClass.superPrototype.method.call().  So maybe this is a useless extension.